internal/engine/wazevo/backend/isa/arm64/cond.go

```diff
@@ -213,3 +213,29 @@ func condFlagFromSSAFloatCmpCond(c ssa.FloatCmpCond) condFlag {
 		panic(c)
 	}
 }
+
+// vecArrangement is the arrangement of data within a vector register.
+type vecArrangement byte
+
+const (
+	// vecArrangementNone is an arrangement indicating no data is stored.
+	vecArrangementNone vecArrangement = iota
+	// vecArrangement8B is an arrangement of 8 bytes (64-bit vector)
+	vecArrangement8B
+	// vecArrangement16B is an arrangement of 16 bytes (128-bit vector)
+	vecArrangement16B
+)
```

Comment:
let's add all of the arrangement now: https://github.com/tetratelabs/wazero/blob/eb11a2caf83e196d318999bee28040637f1be808/internal/asm/arm64/consts.go#L941-L979


and implement formatVRegSizedVec as I suggested ^^